app/models/katello/purpose_status.rb

```diff
@@ -65,7 +65,7 @@ def to_status(options = {})
     end
 
     def relevant?(_options = {})
-      host.subscription_facet.try(:uuid)
+      host.subscription_facet.try(:uuid) && !host.organization.simple_content_access?
```

Comment:
Per my recent comment: https://github.com/Katello/katello/pull/8859#issuecomment-674956576

I think this check is no longer necessary since there should be no cases where we are in SCA mode and there will be any purpose statuses in the db. You can remove it and verify the behavior when registering a new host in SCA mode, changing syspurpose values, and making sure they don't come back in the db, show in the UI, etc. That also means you can remove all of the stubs you added in the Controller tests since that's where `relevant?` is primarily called!

Reply:
Removed this, and confirmed it works the way we want with the manual testing comment. Less changes the better even though this PR has grown quite a bit haha.